src/components/scenes/GettingStartedScene.tsx

```diff
@@ -97,7 +126,7 @@ export const GettingStartedScene = (props: Props) => {
           </TouchableOpacity>
         </Space>
       </SkipButton>
-      <SwipeOffsetDetector swipeOffset={swipeOffset} minOffset={0} maxOffset={sections.length}>
+      <SwipeOffsetDetector swipeOffset={swipeOffset} minOffset={0} maxOffset={sections.length + 1}>
```

Comment:
I realize the `+1` here lacks documentation. Maybe we should factor out the expression to make the code self-documenting. I always default to self-documenting code over adding comments where I can. So, `maxOffset={totalSectionCount}` and `const totalSectionCount = sections.length + 1 // Add one for the final "skip section"` to allow for a clean comment.

Reply:
Or, comment however you like